File: app/src/test/java/net/reichholf/dreamdroid/MinSdkSmokeTest.java
package net.reichholf.dreamdroid;

import org.junit.Test;

import static org.junit.Assert.assertEquals;
import static org.junit.Assert.assertTrue;

public class MinSdkSmokeTest {
    @Test
    public void minSdkIsAtLeast26() {
        assertEquals(26, BuildConfig.MIN_SDK);
        assertTrue(BuildConfig.MIN_SDK >= 26);
    }
}

File: app/androidTest/java/net/reichholf/dreamdroid/MinSdkSmokeTest.kt
package net.reichholf.dreamdroid

import androidx.test.ext.junit.runners.AndroidJUnit4
import org.junit.Assert.assertEquals
import org.junit.Test
import org.junit.runner.RunWith

@RunWith(AndroidJUnit4::class)
class MinSdkSmokeTest {
    @Test
    fun minSdkIs26() {
        assertEquals(26, BuildConfig.MIN_SDK)
    }
}
